### PR TITLE
Feat: implment multicollection search

### PR DESCRIPTION
**Pull request by mbarbet**
Need new contributor to be merged

### CODE REVIEW

File: projects/arlas-toolkit/src/lib/components/search/search-dialog.component.html [hunk 1]
```diff
@@ -16,7 +30,30 @@
...
+                <span class="flex-span">
+                    <mat-icon [matTooltip]="s.collection | getCollectionDisplayName | translate" 
+                    [style.color]="s.color"
+                        class="chip-collection">fiber_manual_record</mat-icon>
+                    <span class="count">{{ s.count }}</span>
+                </span>
+            </div>
+        </div>
+        </li>
+        <li *ngIf="(filteredSearch | async).length === 0" class="search-warning">
```
[the remark this answers is not in the document]
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/src/lib/components/search/search-dialog.component.html:53]: I guess the code under line 30 is when there is one search contributor

**Comment by MohamedHamouGisaia** [comment]
@mbarbet, in configurationUpdater.service, we need to handle the case for searchcontributor in terms of FGA

File: projects/arlas-toolkit/src/lib/components/search/search-dialog.component.html [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
+            <button *ngIf="displayCollectionSettings && searchContributors.length > 1" mat-icon-button
+                [matMenuTriggerFor]="menu">
+                <mat-icon class="settings">settings_outline</mat-icon>
+            </button>
+        </span>
+        <mat-menu #menu="matMenu" class="settings-list">
+            <mat-checkbox #menuItems id="{{ c.label }}" *ngFor="let c of collections;" [(ngModel)]="c.checked"
+                [ngModelOptions]="{standalone: true}" (click)="$event.stopPropagation()" [disableRipple]="true"
+                (change)="onChangeCollection($event)">
+                <span class="flex-span"><mat-icon class="scale-down"
```
[the remark this answers is not in the document]
Comment (mbarbet) [reply projects/arlas-toolkit/src/lib/components/search/search-dialog.component.html:14]: I don't really agree, we're not going to not using Angular tools for fear of migration

File: projects/arlas-toolkit/src/lib/components/search/search-dialog.component.html [hunk 3]
```diff
@@ -16,7 +30,30 @@
...
+    </ul>
+
+    <ul class="autocomplete" *ngIf="searchContributors.length > 1">
+        <li *ngFor="let search of filteredSearch | async" class="search-option"
+            (click)="clickItemSearch(search.key_as_string)">
+            <span class="key">{{ search.key_as_string }}</span>
+            <div class="flex-span collections-count" >
+            <div *ngFor="let s of search.collections" class="flex-span">
+                <span class="flex-span">
+                    <mat-icon [matTooltip]="s.collection | getCollectionDisplayName | translate"
```
[the remark this answers is not in the document]
Comment (mbarbet) [reply projects/arlas-toolkit/src/lib/components/search/search-dialog.component.html:45]: Same response as above

File: projects/arlas-toolkit/src/lib/components/search/search-dialog.component.html [hunk 4]
```diff
@@ -1,13 +1,27 @@
...
+            <mat-checkbox #menuItems id="{{ c.label }}" *ngFor="let c of collections;" [(ngModel)]="c.checked"
+                [ngModelOptions]="{standalone: true}" (click)="$event.stopPropagation()" [disableRipple]="true"
+                (change)="onChangeCollection($event)">
+                <span class="flex-span"><mat-icon class="scale-down"
+                        [style.color]="c.color">fiber_manual_record</mat-icon>
+                    {{ c.label | getCollectionDisplayName | translate }}</span></mat-checkbox>
+        </mat-menu>
         <input matInput [formControl]="searchCtrl" class="full-width-input" autocomplete="off"
-            placeholder="{{searchContributor?.getName() | translate}}" (keyup)="onKeyUp($event)" name="search">
+            placeholder="{{searchContributors[0]?.getName() | translate}}" (keyup)="onKeyUp($event)" name="search">
```
[the remark this answers is not in the document]
Comment (mbarbet) [reply projects/arlas-toolkit/src/lib/components/search/search-dialog.component.html:19]: It's not an input so why do you want a pipe ?

[the remark this answers is not in the document]
Comment (QuCMGisaia) [reply projects/arlas-toolkit/src/lib/components/search/search-dialog.component.html:19]: To remove the getName from the template

**Pull request merged**